marlowe-runtime-cli/app/Language/Marlowe/Runtime/CLI/Command/Create.hs

```diff
@@ -249,7 +250,7 @@ runCreateCommand TxCommand{walletAddresses, signingMethod, tagsFile, metadataFil
         else do
           hPutStrLn stderr "Safety analysis found the following errors in the contract:"
           BS8.hPutStrLn stderr $ Yaml.encode safetyErrors
-    liftIO . print $ A.encode (A.object [("contractId", toJSON . renderTxOutRef $ contractId)])
+    liftIO . putStrLn . T.unpack . renderTxOutRef $ contractId
```

Comment:
This will break the tutorials, including the starter kit, so we'll need to update those.

Reply:
Another accident :-(